src/v2/Components/v2/ArtworkFilter/ArtworkFilters/ResultsFilter.tsx

```diff
@@ -14,10 +14,30 @@ export const sortResults = (
   selectedValues: Array<string>,
   allResults: Array<Result>
 ) => {
+  // sort all results by `count` in descending order
+  allResults.sort((a, b) => {
+    if (a.count < b.count) {
+      return 1
+    } else if (a.count === b.count) {
+      // sort by `name` in ascending order if `count` is equal
+      if (a.name > b.name) {
+        return 1
+      } else if (a.name === b.name) {
+        return 0
+      } else {
+        return -1
+      }
+    } else {
+      return -1
+    }
+  })
```

Comment:
I wrote this compare function because results with the same `count` were being listed in reverse-alphabetic order when we sorted the results by `count` and reversed the order:

```js
sortBy(results, ["count"]).reverse()
```

Reply:
I would probably try to avoid the mutation since in general I find that pretty surprising to see.

Would also consider just flatting out the conditional since this is pretty difficult to read?

Reply:
I think you were referring to the `allResults.sort()` operation when you suggested avoiding the mutation. I agree that it would be bad to change that array, so I've updated it to sort on a copy of that array. Thanks for pointing that out.

I took a shot at flattening out that conditional. Please let me know what else you may change.

Reply:
👍  and yeah much more legible IMO

Reply:
interesting! i think you can use `lodash` to make it a little bit more legible:

```
orderBy(results, ['count','name'], ['desc', 'asc'])
```

if we wanted to be exact with that.

Reply:
That worked! I was so focused on `sortBy` that I didn't look at what else Lodash has to offer. Thanks for teaching me about that util.